jormungandr/src/leadership/process.rs

```diff
@@ -536,6 +536,7 @@ impl Module {
         }
     }
 
+    #[allow(clippy::comparison_chain)]
```

Comment:
As the lint suggests, this could be rewritten as a match over `epoch_tip.cmp(current_slot_position.epoch)`. The compiler will catch if the match is non-exhaustive if this code is later changed.